canonical/canonical.com
# Clean up custom styles

## Done

- Removed any custom styles that are not needed or are currently not being used
- Drive by: 
  - Update strip usage on /partners/executive-summit 

## Note

There are still some styles to remove here, specifically those used in templates that no longer belong on the site. To not bloat this PR too much I will remove these in a separate PR along with custom styles they use 

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: https://canonical-com-968.demos.haus/
- Click through the site and make sure nothing is broken

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-4915
